diff --git a/node_modules/@hummer/tenon-style-loader/dist/helper/style.js b/node_modules/@hummer/tenon-style-loader/dist/helper/style.js
index 97b55eb..f8f7e51 100644
--- a/node_modules/@hummer/tenon-style-loader/dist/helper/style.js
+++ b/node_modules/@hummer/tenon-style-loader/dist/helper/style.js
@@ -50,11 +50,7 @@ function generateCode(ruleSetMap, options) {
     var options = ${JSON.stringify(options)};
   `;
     return `
-    import {collectStyle} from '${packageName}';
-    export default (function(){
-      ${styleCode}
-      return collectStyle(ruleSetMap, options);
-    })();
+
   `;
 }
 function getRuleStyle(node) {
